src/routes/App.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+
+import LoginPage from '../component/Pages/LoginPage.jsx';
+import Homepage from '../component/Pages/HomePage.jsx';
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
```

Comment:
Move your routes into a separate file.
This App should be quite simple right now